tasks/test_results_processor.py

```diff
@@ -150,6 +182,12 @@ def _bulk_write_tests_to_db(
                 )
             )
 
+            test_flag_bridge_data += [
+                {"test_id": test_id, "repoid": repoid, "flag_id": repo_flags[flag]}
```

Comment:
Because test_id is not dependent on flag in flags I think we can move that outside of the list comprehension and reduce some cycles spent evaluating that for every flag right?

Reply:
at some point we'd have to turn it into a list of dicts with each value so sqlalchemy can insert it